ext/src/dll/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ set(OPENTELEMETRY_EXPORT_DEF
 
 add_library(opentelemetry_cpp SHARED dllmain.cc ${OPENTELEMETRY_EXPORT_DEF})
 
+if(WITH_STL)
+  add_compile_definitions(WITH_STL)
+endif()
+
```

Comment:
Please take a look at: https://github.com/open-telemetry/opentelemetry-cpp/blob/main/api/CMakeLists.txt#L54-L84

WITH_STL is no longer an ON/OFF flag.

Reply:
You're right, corrected this in the latest commit to ``add_compile_definitions(OPENTELEMETRY_STL_VERSION)`` to cover the case of building WITH_STL and C++14.